src/libs/Environment/Environment.ts

```diff
@@ -47,11 +48,18 @@ function getEnvironmentURL(): Promise<string> {
     });
 }
 
+/**
+ * Given the environment get the corresponding oldDot URL
+ */
+function getOldDotURLFromEnvironment(environment: Environment): string {
+    return OLDDOT_ENVIRONMENT_URLS[environment];
+}
+
```

Comment:
We already have getOldDotEnvironmentURL so we don't need this right?

Reply:
Yeah but it returns a Promise. We could make it work but I thought using this where we provide the environment variable lets us do it synchronously and prevents us from having to do an async call in the modal

Reply:
bump @thienlnam

Reply:
Sorry, forgot to get back to this - I guess that is true, it just feels odd having the same method twice and I imagine people might not know which one to use when they see this. But alternatively I guess you'd have to use a useEffect to get the baseURL so this is cleaner